app/build.gradle

```diff
@@ -79,11 +79,8 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
 
     //User Item
-    implementation 'com.google.firebase:firebase-storage:19.2.0'
-    implementation 'com.google.firebase:firebase-auth:19.3.2'
     implementation 'com.google.firebase:firebase-database:19.4.0'
     implementation 'junit:junit:4.12'
-    implementation 'com.firebaseui:firebase-ui-storage:6.2.0'
     implementation 'com.github.bumptech.glide:glide:4.11.0'//adding glid library
```

Comment:
@enavhasson ui-storage and glide exists as well, I'll update it in the next PR :)